deploy/bootstrap/.env

```diff
@@ -0,0 +1,2 @@
+GOSSIP_PEERS=ws://bliny.nodes.orbs-test.com:60001,ws://pelmeni.nodes.orbs-test.com:60001,ws://borscht.nodes.orbs-test.com:60001,ws://pirogi.nodes.orbs-test.com:60001,ws://oladyi.nodes.orbs-test.com:60001,ws://olivier.nodes.orbs-test.com:60001,ws://solyanka.nodes.orbs-test.com:60001
```

Comment:
Can you write these as multilines, for improved readability?

Reply:
I'm not sure it's a very good idea for an `.env` file, we might want to parse it somewhere from scratch and newlines might not be supported.

Reply:
Alright.